examples/player/simid_player.html

```diff
@@ -23,15 +23,30 @@
     }
   </style>
   <script>
+    //if ad is non-linear allow user to insert specific dimensions
+    function enableDisable() {
```

Comment:
enable/disable what?

Also, imo use toggle instead of enableDisable

Reply:
changed this function in button PR